lib/Notes/utils/getHTMLSubstring.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Returns a substring of the required length with HTML tags
+ *
+ * @param {string} htmlString string with HTML elements
+ * @param {number} maxLength required max length of string
+ */
+export default function getHTMLSubstring(htmlString, maxLength) {
+  const element = document.createElement('div');
+  const fragment = document.createDocumentFragment();
+  let stringLength = 0;
+
+  element.innerHTML = htmlString;
+
+  function recurse(currentNode) {
+    const nodeValue = currentNode.nodeValue;
+    const clonedNode = currentNode.nodeType === 1
```

Comment:
Please use a constant that describes this magic number

Reply:
Done